### PR TITLE
Fix: Removed licenses step from action

### PR DESCRIPTION
**Pull request by agg3fe**
## Description
Fixing broken pipe line issue after adding mandatory new licenses

### CODE REVIEW

File: .github/workflows/publish-image-registry.yml [hunk 1]
```diff
@@ -62,19 +62,6 @@ jobs:
       - name: install dependencies
         working-directory: ./.github/actions/generate-dependencies-notice
         run: npm install
-      - name: baselayer-licenses
-        uses: philips-labs/tern-action@v1.1.0
-        with:
-          image: eclipse-temurin:17-jre-alpine
-          format: json
-          output: ./base-image-layers.json
-      - name: dependency-licenses
```
Comment (tunacicek) [inline .github/workflows/publish-image-registry.yml:71]: Why is this task removed? 
We need dependencies-notice

**Comment by tunacicek** [comment]
@agg3fe : If the steps for licenses are removed, we need to find a way to copy the required file into the Docker image. For example in the dockerfile.

**Review by tunacicek — CHANGES_REQUESTED** [review]
If the steps for licenses are removed, we need to find a way to copy the required file into the Docker image. For example in the dockerfile.

**Comment by agg3fe** [comment]
> @agg3fe : If the steps for licenses are removed, we need to find a way to copy the required file into the Docker image. For example in the dockerfile.

@tunacicek PR is updated.

**Review by tunacicek — APPROVED** [review]
LGTM

**Pull request merged**